Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor

```diff
@@ -1,6 +1,96 @@
 ﻿@page "/tabs";
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Tabs</h2>
+        <p>
+            Tabs are components that help you organize and navigate multiple documents in a single container. They can be used for switching between items in the container.
+        </p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            There are two pieces to a tabbed interface: the tabs themselves, and the content for each tab. The tabs are an component with the <Code Content="PFRhYkhlYWRlcj4="></Code>, and each item has the <Code Content="PFRhYkhlYWRlclRpdGxlPg=="></Code>. Each tab contains a link to a tab.
+        </p>
+        <p>
+            The tab content container has the <Code Content="PFRhYkNvbnRlbnRXcmFwcGVyPg=="></Code>, while each section has the <Code Content="PFRhYkNvbnRlbnQ+"></Code>. Each content pane also has a unique ID, which is targeted by a link in the tabstrip.
```

Comment:
The tab content container has the ????? the what????
while each section has the ??????? same c'mon man!

Reply:
They’re in the Code Content

Reply:
The English did not make sense